src/Npgsql/NpgsqlEventSource.cs

```diff
@@ -96,7 +96,7 @@ internal void MultiplexingBatchSent(int numCommands, Stopwatch stopwatch)
         }
 
 #if !NETSTANDARD2_0
-        int GetIdleConnections()
+        double GetIdleConnections()
```

Comment:
Is there a specific reason for these methods to return double? It shouldn't matter if the cast happens inside them or outside.. (doesn't matter either way)

Reply:
Only a small one - that way we can just pass the delegate to the counter (`new PollingCounter("connection-pools", this, GetPoolsCount)` instead of `new PollingCounter("connection-pools", this, () => GetPoolsCount)`).

Reply:
Sure, makes sense!